src/function-builder.ts

```diff
@@ -347,14 +376,10 @@ export class FunctionBuilder {
       ) => https._onRequestWithOptions(handler, this.options),
       /**
        * Declares a callable method for clients to call using a Firebase SDK.
+       * @param opts Configuration options for a callable function.
        * @param handler A method that takes a data and context and returns a value.
        */
-      onCall: (
-        handler: (
-          data: any,
-          context: https.CallableContext
-        ) => any | Promise<any>
-      ) => https._onCallWithOptions(handler, this.options),
+      onCall: _onCall.bind(this),
```

Comment:
I'm not sure I feel comfortable with a free-floating function that has a "this" method that must be used with "bind". Why can't this just be an instance method?